### DISCUSSION TITLE
After long working hours users without grant permissions receives this message

### DISCUSSION THREAD

**Discussion by kilian099** [kimai/kimai#4641] [General]
### Describe the issue

Chrome Browser

![image](https://github.com/kimai/kimai/assets/106171763/f4a9f9d4-7121-4e6e-9ae4-6595b2e8ce71)

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

latest

### How do you run Kimai?

Shared-Hosting

### Which PHP version are you using?

8.1

**Comment by kevinpapst** [comment]
This looks like Session timeout, which is an environment setting. 
Default values of PHP are 14 minutes.
Try to set the session timeout to a week.

What do you mean by "without grant permissions"?

**Reply by kilian099** [reply]
Hi @kevinpapst , I meant with grant permissions. A user who doesn't have privilege permissions is a simple user with a Team Lead role. I will try to change the permissions on the PHP.ini file in the Hosting server and get back to you if there are any issues.
